buildlib/azure-pipelines-release.yml

```diff
@@ -102,9 +102,23 @@ stages:
     condition: eq(dependencies.Prepare.outputs['CheckRelease.Result.Launch'], 'True')
     jobs:
       - template: az-distro-release.yml
-      - template: jucx/jucx-publish.yml
```

Comment:
it seems like nothing triggers jucx-publish.yml step now..

Reply:
Lost in splitting. Thanks for noticing!